{
  try {
    marshaller.connect();
  }
 catch (  final IOException e) {
    throw new ConnectionException("Connection failure",e);
  }
}
